cpp/src/parquet/properties.h

```diff
@@ -789,6 +857,19 @@ class PARQUET_EXPORT WriterProperties {
     return false;
   }
 
+  bool bloom_filter_enabled() const {
```

Comment:
yeah, I'm not sure I would include this as an option it seems like it really should be path specific.

Reply:
This can be refered as "any bloom filter enabled" as a fastpath for skip checking?